great_tables/_options.py

```diff
@@ -969,3 +969,673 @@ def opt_all_caps(
         res = tab_options(res, row_group_text_transform="inherit")
 
     return res
+
+
+def opt_stylize(self: GTSelf, style: int = 1, color: str = "blue") -> GTSelf:
+    """
+    Stylize your table with a colorful look.
+
+    With the `opt_stylize()` method you can quickly style your table with a carefully curated set of
+    background colors, line colors, and line styles. There are six styles to choose from and they
+    largely vary in the extent of coloring applied to different table locations. Some have table
+    borders applied, some apply darker colors to the table stub and summary sections, and, some even
+    have vertical lines. In addition to choosing a `style` preset, there are six `color` variations
+    that each use a range of five color tints. Each of the color tints have been fine-tuned to
+    maximize the contrast between text and its background. There are 36 combinations of `style` and
+    `color` to choose from.
+
+    Parameters
+    ----------
+    style : int
+        Six numbered styles are available. Simply provide a number from `1` (the default) to `6` to
+        choose a distinct look.
+    color : str
+        The color scheme of the table. The default value is `"blue"`. The valid values are `"blue"`,
+        `"cyan"`, `"pink"`, `"green"`, `"red"`, and `"gray"`.
+
+    Returns
+    -------
+    GT
+        The GT object is returned. This is the same object that the method is called on so that we
+        can facilitate method chaining.
+
+    Examples
+    --------
+    Using select columns from the `exibble` dataset, let's create a table with a number of
+    components added. Following that, we'll apply a predefined style to the table using the
+    `opt_stylize()` method.
+
+    ```{python}
+    from great_tables import GT, exibble, md
+
+    gt_tbl = (
+          GT(
+            exibble[["num", "char", "currency", "row", "group"]],
+            rowname_col="row",
+            groupname_col="group"
+          )
+          .tab_header(
+            title=md("Data listing from **exibble**"),
+            subtitle=md("`exibble` is a **Great Tables** dataset.")
+          )
+          .fmt_number(columns="num")
+          .fmt_currency(columns="currency")
+          .tab_source_note(source_note="This is only a subset of the dataset.")
+          .opt_stylize()
+        )
+
+    gt_tbl
+    ```
+
+    The table has been stylized with the default style and color. The default style is `1` and the
+    default color is `"blue"`. The default style is a combination of color and border settings that
+    are applied to the table. The default style and color can be modified by providing the `style=`
+    and `color=` arguments.
+
+    ```{python}
+    gt_tbl.opt_stylize(style=2, color="green")
+    ```
+    """
+
+    # Validate the `style` and `color` arguments
+    if style not in [1, 2, 3, 4, 5, 6]:
```

Comment:
Pairing, I asked why we use numbers and not names for styles. Rich and I copied out all the 6 styles side-by-side, and I just want to +1 that it's very hard to think of names that would distinguish these things.